docs/src/model_creation/programmatic_CRN_construction.md

```diff
@@ -1,251 +0,0 @@
-# [Programmatic Construction of Symbolic Reaction Systems](@id programmatic_CRN_construction)
```

Comment:
The difference between this and the new file too large for displaying them side by side is not helpful. I have no opinions on filename though, just used a new one to differentiate the new file.